### PR TITLE
simple slack notifier

### PR DESCRIPTION
**Pull request by AlexFinn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,6 +246,19 @@ CREATE TABLE checks (name TEXT NOT NULL PRIMARY KEY, timestamp INT8, results TEX
 
 Currently the status page does not support SQL storage.
 
+#### Slack notifier
+
+slack configuration:
```
Comment (mholt) [inline README.md]: "Enable notifications in Slack with this Notifier configuration:"

File: README.md [hunk 2]
```diff
@@ -242,6 +246,19 @@ CREATE TABLE checks (name TEXT NOT NULL PRIMARY KEY, timestamp INT8, results TEX
...
+```json
+{
+	"name": "slack",
+	"username": "username",
+	"channel": "#channel-name",
+	"webhook": "webhook-url"
+}
+```
+
+How to [create webhook](https://get.slack.help/hc/en-us/articles/115005265063-Incoming-WebHooks-for-Slack)
```
Comment (mholt) [inline README.md]: Use a full sentence, like "Follow these instructions to create a webhook."

**Comment by AlexFinn** [comment]
@mholt  thanks for the tips

File: README.md (hunk 2 above)
Comment (AlexFinn) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (AlexFinn) [reply README.md]: done

**Comment by mholt** [comment]
Thanks! Neat addition.

**Pull request merged**